services/sentry/lib/getPayout.js

```diff
@@ -1,18 +1,71 @@
+/* eslint-disable no-nested-ternary */
+// const BN = require('bignumber.js') // allows
 const BN = require('bn.js')
 const toBalancesKey = require('../toBalancesKey')
 
 function getPayout(channel, ev) {
 	if (ev.type === 'IMPRESSION' && ev.publisher) {
 		// add the minimum price for the event to the current amount
-		return [toBalancesKey(ev.publisher), new BN(channel.spec.minPerImpression || 1)]
+		const minPrice = new BN(channel.spec.minPerImpression || 1)
```

Comment:
this should use `pricingBounds` and only fallback to minPerImpression/maxPerImpression